packages/app/src/app/pages/Sandbox/Editor/Header/SandboxName/SandboxName.tsx

```diff
@@ -19,136 +25,132 @@ import {
-        ? basename(currentSandbox.collection.path) ||
-          (currentSandbox.team ? currentSandbox.team.name : 'My Sandboxes')
-        : 'My Sandboxes';
+  const folderName = currentSandbox.collection
+    ? basename(currentSandbox.collection.path) ||
+      (currentSandbox.team ? currentSandbox.team.name : 'My Sandboxes')
+    : 'My Sandboxes';
 
-      const template = currentSandbox.customTemplate && !updatingName;
-      const spring = useSpring({
-        opacity: updatingName ? 0 : 1,
-        pointerEvents: updatingName ? 'none' : 'initial',
-      });
-      const { customTemplate, owned } = currentSandbox;
+  const template = currentSandbox.customTemplate && !updatingName;
+  const spring = useSpring({
+    opacity: updatingName ? 0 : 1,
+    pointerEvents: updatingName ? 'none' : 'initial',
+  });
+  const { customTemplate, owned } = currentSandbox;
 
-      return (
-        <Main>
-          <Container>
-            {!customTemplate && owned && (
-              <animated.div style={spring}>
-                <Folder>
-                  {isLoggedIn ? (
-                    <FolderName
-                      onClick={() => modalOpened({ modal: 'moveSandbox' })}
-                    >
-                      {folderName}
-                    </FolderName>
-                  ) : (
-                    'Anonymous '
-                  )}
-                  /{' '}
-                </Folder>
-              </animated.div>
-            )}
-            {updatingName ? (
-              <Form onSubmit={submitNameChange}>
-                <NameInput
-                  autoFocus
-                  innerRef={(el: HTMLInputElement) => {
-                    if (el) {
-                      el.focus();
-                    }
-                  }}
-                  onKeyUp={handleKeyUp}
-                  onBlur={handleBlur}
-                  onChange={handleInputUpdate}
-                  placeholder={name}
-                  value={value}
-                />
-              </Form>
-            ) : (
-              <Name owned={owned} onClick={owned ? handleNameClick : null}>
-                {sandboxName}
-              </Name>
-            )}
-            {template ? (
-              <Tooltip
-                interactive
-                delay={0}
-                placement="bottom"
-                content={
-                  <>
-                    This sandbox is a template, you can learn about templates in
-                    the{' '}
-                    <Link target="_blank" to="/docs/templates">
-                      docs
-                    </Link>
-                    .
-                  </>
+  return (
+    <Main>
+      <Container>
+        {!customTemplate && owned && (
+          <animated.div style={spring}>
+            <Folder>
+              {isLoggedIn ? (
+                <FolderName
+                  onClick={() =>
+                    modalOpened({ message: null, modal: 'moveSandbox' })
```

Comment:
@christianalfoni Too bad `message` needs to be passed 😕
Maybe make it optional? 🤔